.gitignore

```diff
@@ -1,10 +1,13 @@
 node_modules/
-amd-test/browser-source-map-support.js
+browser-source-map-support.js
 amd-test/script.js
 amd-test/script.map
+amd-test/script.js.map
 browserify-test/compiled.js
-browserify-test/script.js
-browserify-test/script.map
 header-test/script.js
 header-test/script.map
-webpack-test/compiled.js
\ No newline at end of file
+header-test/script.js.map
+webpack-test/compiled.js
+browser-test/script.js
+browser-test/script.map
+browser-test/script.js.map
```

Comment:
Ignore files which creates by prepare script

Reply:
I think that these files are intentionally checked in so that one can depend on versions directly from github, download the files directly from github, etc.

Reply:
Sorry, probably I didn't get it.
In my opinion we don't have to store in GitHub such files as generated during build/prepare or etc
This files generated due prepare script and we use they only in tests and I really don't see any reason to keep they in repo

Reply:
I generally agree, need to take extra care so we don't break anything for anyone though 🤔